Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.14.13
```

Comment:
Noting version is different than what is specified in go.mod. I should change this